specification/iotcentral/data-plane/Microsoft.IoTCentral/stable/2022-07-31/iotcentral.json

```
@@ -0,0 +1,6113 @@
+    "/scheduledJobs/{scheduledJobId}": {
+      "get": {
+        "operationId": "ScheduledJobs_Get",
+        "summary": "Get a scheduled job by ID",
+        "description": "Get details about a scheduled job by ID.",
+        "x-ms-examples": {
+          "List jobs": {
+            "$ref": "./examples/scheduledjobs_get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "$ref": "#/parameters/ScheduledJobId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ScheduledJob"
+            }
+          },
+          "default": {
+            "description": "An error response received from the IoT Central Service.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "ScheduledJobs_Create",
+        "summary": "Create or update a scheduled job",
+        "description": "Create or update a scheduled job by ID.",
+        "x-ms-examples": {
+          "List jobs": {
+            "$ref": "./examples/scheduledjobs_create.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/Version"
+          },
+          {
+            "$ref": "#/parameters/ScheduledJobId"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Scheduled job definition.",
+            "schema": {
+              "$ref": "#/definitions/ScheduledJob"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ScheduledJob"
+            }
+          },
+          "default": {
+            "description": "An error response received from the IoT Central Service.",
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "Error code for specific error that occurred."
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "patch": {
+        "operationId": "ScheduledJobs_Update",
+        "summary": "Update a scheduled job via patch",
+        "description": "Update an existing scheduled job by ID.",
+        "x-ms-examples": {
+          "List jobs": {
+            "$ref": "./examples/scheduledjobs_update.json"
+          }
+        },
+        "consumes": [
+          "application/json",
+          "application/merge-patch+json"
```

Patch should only accept "application/merge-patch+json"
```suggestion
          "application/merge-patch+json"
```

---

fixed